linux/net/rina/ipcps/shim-eth-vlan.c

```diff
@@ -51,6 +51,8 @@
 #define DEFAULT_QDISC_MAX_SIZE 50
 #define DEFAULT_QDISC_ENABLE_SIZE 10
 
+#define BLOCKING_FLOW 1
```

Comment:
Can you elaborate why this is needed? Is this needed in every shim? Then probably it should be defined in a common place (common.h)?